README.md

```diff
@@ -35,28 +41,75 @@ Before you can run the bot yourself, you'll need to set up an application on Dis
 
 ### Configuring
 
-High Quality Bot expects a file called `config.cfg` to be in the same directory as the .jar file, and won't run without it. The format is simply one `key=value` declaration per line. Currently available options to set are:
+High Quality Bot expects a file called `config.cfg` to be in the same directory as the .jar file, and won't run without
+it. The format is simply one `key=value` declaration per line. `exampleconfig.cfg` provides a sample configuration.
+Currently available options to set are:
 * `token` - Your Bot account's OAuth token
 * `prefix` - The desired character (or string of characters if you're a crazy person) used to invoke commands.
+* `ownerid` - The User attached to this ID is automatically given permission to use all commands.
+* `allowed` - This is a list of command names separated by commas that users are allowed to use by default.
+* `disallowed` - Users are by default explicitly disallowed to use these commands.
+* `allowedHasPrecedence` - If true, any commands in both `allowed` and `disallowed` are considered allowed.
```

Comment:
My thinking was that this allows users to avoid having to create a bunch of extra roles. So, if a "general" role allows access to things like fun commands, a mod role would only need to add on administrative commands on top of that general role. It allows for a setup where each role only needs to explicitly give or deny permission for what is relevant to that role, and not care about anything else